grid2op/tests/test_Opponent.py

```diff
@@ -9,7 +9,7 @@
 import tempfile
 import warnings
 from grid2op.tests.helper_path_test import *
-from grid2op.Opponent import BaseOpponent, RandomLineOpponent
+from grid2op.Opponent import BaseOpponent, NeuripsOpponent
```

Comment:
Wait i'm scared now.

Put back all the tests we did before, when we validate the opponent (for version 1.0.0) and then add some tests for NeuripsOpponent.
Don't change the tests like that.

tests are added, it's extremely rare that you removed them.

Reply:
done